### PR TITLE
feat: ✨ add getMyThreads

### PR DESCRIPTION
**Pull request by paychorn**

### CODE REVIEW

File: thread-service/src/server.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { z } from "zod";
 
 import { getAuthenticatedUserId } from "../../user-service/src/libs/token";
 import { applyAnonymity, sanitizeThreadRequest } from "./decorator";
-import { Empty, SearchQuery, ThreadId, ThreadList } from "./models";
+import { Empty, SearchQuery, ThreadId,  ThreadList } from "./models";
```
Comment (ThanatWongsamut) [inline thread-service/src/server.ts]: Just format

**Review by ThanatWongsamut — COMMENTED** [review]
Just need to format

File: thread-service/src/server.ts (hunk 1 above)
Comment (paychorn) [reply thread-service/src/server.ts]: done 🐥

**Review by ThanatWongsamut — APPROVED** [review]
Nice

**Pull request merged**